sakura_core/macro/CSMacroMgr.h

```diff
@@ -130,7 +130,7 @@ class CSMacroMgr
 	{
 		return ( 0 <= idx && idx < MAX_CUSTMACRO ) ?
 		m_pShareData->m_Common.m_sMacro.m_MacroTable[idx].m_szFile : 
-		( (idx == STAND_KEYMACRO || idx == TEMP_KEYMACRO) && m_sMacroPath != L"" ) ?
+		( (idx == STAND_KEYMACRO || idx == TEMP_KEYMACRO) && !m_sMacroPath.empty() ) ?
```

Comment:
```suggestion
		( (idx == STAND_KEYMACRO || idx == TEMP_KEYMACRO) && m_sMacroPath.length() ) ?
```

Reply:
@berryzplus fix it https://github.com/sakura-editor/sakura/pull/1860/commits/1fc1801a340c0d1ebb2b306d27b3d48c14ec2c17